src/analyzer.ts

```diff
@@ -335,7 +335,10 @@ export class Analyzer {
     const scriptCommand = syntaxInfo.scriptNode;
     const wireitConfig = syntaxInfo.wireitConfigNode;
 
-    if (wireitConfig !== undefined && scriptCommand.value !== 'wireit') {
+    if (
+      wireitConfig !== undefined &&
+      ![`wireit`, `yarn run -T wireit`].includes(scriptCommand.value)
```

Comment:
Seems like `yarn run -T -B wireit` would be good (though mind verifying that works?), since that enforces it is a binary, and not a local script called `wireit`?